test/expected.txt

```diff
@@ -46,6 +47,13 @@
 || 
 || 0 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  about]
+|| This is a CLI (Command Line Interface) Address Book application written in OOP fashion.
```

Comment:
The word "this" is ambiguous. Consider using "This application" instead.

Reply:
Thanks for bringing this up Sean. Wording is taken and preserved from the Readme.md of the upstream repo and kept for consistency.